packages/rest/src/router/openapi-path.ts

```diff
@@ -62,5 +62,7 @@ export function getPathVariables(path: string) {
  * @param path OpenAPI path with optional variables as `{var}`
  */
 export function toExpressPath(path: string) {
-  return path.replace(POSSIBLE_VARNAME_PATTERN, ':$1');
+  // Convert `.` to `\\.` so that path-to-regexp will treat it as the plain
+  // `.` character
+  return path.replace(POSSIBLE_VARNAME_PATTERN, ':$1').replace('.', '\\.');
```

Comment:
just curious: why `.` is special here? I realized that we don't do the conversion for other special characters like `@`.

Reply:
`.` is special char for regexp.